src/libs/conduit/conduit_utils.hpp

```diff
@@ -21,12 +21,16 @@
 #include <sstream>
 #include <chrono>
 
-
 //-----------------------------------------------------------------------------
 // -- conduit includes --
 //-----------------------------------------------------------------------------
 #include "conduit_core.hpp"
 
+//-----------------------------------------------------------------------------
+#if defined(CONDUIT_USE_CALIPER)
+#include <caliper/cali.h>
+#endif
+
```

Comment:
Is conduit_utils.hpp a user-facing header file?  Will this let caliper leak out into a host code through Conduit?

Reply:
that one I need to remove, good catch!